### ISSUE TITLE
 Cannot create property 'url' on string 'en'

### ISSUE THREAD

**Issue by akosyakov** [redhat-developer/vscode-yaml#556]
Create .gitpod.yml file with any content like:
```
image:
  file: .gitpod.Dockerfile
ports:
  - port: 3000
    onOpen: open-browser
tasks:
  - init: |
      yarn
      yarn --cwd ./build compile
      yarn compile
      yarn download-builtin-extensions
    command: |
      gp sync-done init
      export NODE_ENV=development
      export VSCODE_DEV=1
      yarn gulp watch-init
    name: watch app
  - command: |
      export NODE_ENV=development
      export VSCODE_DEV=1
      gp sync-await init
      yarn gulp watch-extension:gitpod
    name: watch extension
  - command: |
      export NODE_ENV=development
      export VSCODE_DEV=1
      gp sync-await init
      node out/gitpod.js
    name: run app
    openMode: split-right
github:
  prebuilds:
    pullRequestsFromForks: true
vscode:
  extensions:
    - dbaeumer.vscode-eslint
```

Extension keeps failing with:
```
[2021-07-14 09:04:25.802] [renderer1] [error] Request textDocument/hover failed with message: Cannot create property 'url' on string 'en': Error: Request textDocument/hover failed with message: Cannot create property 'url' on string 'en'
    at handleResponse (/Users/akosyakov/.vscode/extensions/redhat.vscode-yaml-0.21.1/node_modules/vscode-jsonrpc/lib/common/connection.js:477:48)
    at processMessageQueue (/Users/akosyakov/.vscode/extensions/redhat.vscode-yaml-0.21.1/node_modules/vscode-jsonrpc/lib/common/connection.js:292:17)
    at Immediate.<anonymous> (/Users/akosyakov/.vscode/extensions/redhat.vscode-yaml-0.21.1/node_modules/vscode-jsonrpc/lib/common/connection.js:276:13)
    at processImmediate (internal/timers.js:461:21)```

**Comment by akosyakov** [comment]
https://gitpod.io/schemas/gitpod-schema.json does not look really good, so probably an issue on Gitpod side, investigating

**Comment by fbricon** [comment]
Should be fixed with vscode-yaml 0.22.0
